lib/jackal-slack/notification.rb

```diff
@@ -14,7 +14,9 @@ def setup(*_)
       # @return [Truthy, Falsey]
       def valid?(message)
         super do |payload|
-          payload.get(:data, :slack)
```

Comment:
But since the messages key is used within execution, why would the check for that be wrong?
I was thinking of a case where messages is nil and then it erroring when trying to iterate.

Or is the assumption that if data -> slack is present, messages should def. be there?